code/game/machinery/embedded_controller/airlock_controllers.dm

```diff
@@ -13,10 +13,14 @@
 	var/tag_shuttle_mech_sensor
 	var/tag_secure = 0
 
-/obj/machinery/embedded_controller/radio/airlock/Initialize()
-	..()
+/obj/machinery/embedded_controller/radio/airlock/Initialize(mapload)
+	. = ..()
 	program = new/datum/computer/file/embedded_program/airlock(src)
```

Comment:
(Unrelated to the PR)

These need to die. This airlock program is one of the most intensive things on the server.
![image](https://user-images.githubusercontent.com/25063394/180738202-ea2ea637-0dbf-47d5-8e29-f618a3c50a8a.png)

**3 ENTIRE MILLISECONDS TO TELL A VENT TO COME ON. THATS 12% OF THE ENTIRE TICK!!! ON JUST THIS!!**